instock_connector/README.md

```diff
@@ -30,19 +30,68 @@ There are two ways for installing the connector Python package.
 
 1. From PyPi: `pip install inorbit-instock-connector`
 
-2. From source: clone the repository and run `pip install -r instock_connector/requirements.txt`
+2. From source: clone the repository and install the dependencies:
+
+```bash
+cd instock_connector/
+virtualenv .venv
+. .venv/bin/activate
+pip install -r requirements.txt
+```
 
 Configure the Connector:
 
-- Copy [config/myinstock.example.yaml](config/myinstock.example.yaml) and modify the settings to match your setup.
+- Copy [`config/myinstock.example.yaml`](config/myinstock.example.yaml) and modify the settings to match your setup.
 
-- Copy `config/example.env` to `config/.env` and set the environment variables following the instructions in the same
+- Copy [`config/example.env`](config/example.env) to `config/.env` and set the environment variables following the instructions in the same
   file. You can get the `INORBIT_KEY` for your account from InOrbit's
   [Developer Console](https://developer.inorbit.ai/docs#configuring-environment-variables).
 
 - Also apply the Configuration as Code manifests under the [cac_examples](cac_examples) folder, through the
   [InOrbit CLI](https://developer.inorbit.ai/docs#using-the-inorbit-cli).
 
+## Deployment
+
+### Run the Connector manually
+
+Once all dependencies are installed and the configuration is complete, the Connector can be run as a Python script. The entry point is `inorbit_instock_connector/src/main.py`.
+
+```bash
+# Add the environment variables, activate the virtual environment and run the Connector
+cd instock_connector/
+export $(grep -v '^#' config/.env | xargs) && \
+source .venv/bin/activate && \
+python inorbit_instock_connector/src/main.py --help
```

Comment:
It was just a suggestion, since I didn't feel the need to explain how to use it. I changed it with an example and suggested using `--help` in the paragraph above, which I think is better.